infra.bs

```diff
@@ -216,8 +217,11 @@ in parentheses. [[!UNICODE]]
 
```

Comment:
We acknowledge them for code point explicitly and everything else builds on top of that. I'll reference them again for code unit though as you suggested.